### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by SuperQ**
* [FEATURE] Add TLS and basic authentication #101

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ make
 Alternatively a Dockerfile is supplied:
 
 ```sh
-docker run -p 9150:9150 quay.io/prometheus/memcached-exporter:v0.7.0
+docker run -p 9150:9150 quay.io/prometheus/memcached-exporter:latest
```
Comment (grobie) [inline README.md:24]: Should we add a note on top explaining that this tag returns the current main branch version, and that people should check quay/docker hub for the list of currently available tags?

**Review by grobie — APPROVED** [review]
:+1: in general. I'd just leave a note in the README to make it clear that the example will result in a potentially less stable version.

File: README.md (hunk 1 above)
Comment (SuperQ) [reply README.md:24]: This returns the latest current release, not the main branch. We publish that under `:master`.

Comment (grobie) [reply README.md:24]: Oh, even better then!

**Pull request merged**